packages/adelus/example/Makefile

```diff
@@ -0,0 +1,59 @@
+KOKKOS_PATH = ${HOME}/Trilinos/Trilinos.fork/packages/kokkos
```

Comment:
@srajama1 I left the Makefile there just in case there is any user application still uses Makefile. However, it looks like only Gemma will be using Adelus, so I removed the Makefile.